lib/registration.pm

```diff
@@ -440,7 +440,12 @@ sub fill_in_reg_server {
     }
     else {
         send_key "alt-i";
-        send_key "alt-o";
+        if (sle_version_at_least('15')) {
```

Comment:
probably it will be more correct to add `&& is_sle()` ?

Reply:
Maybe dumb question, but `sle_version_at_least` should not include `is_sle`? For me, and due to the beginning of the function name, it is not supposed to return something on a openSUSE for example.

Reply:
@okurz  can you please answer this :)

Reply:
@asmorodskyi  I have consider about it it the first place. But I think it is no need to add is_sle, because the registration.pm only be used for SLES case.

Reply:
@ldevulder you might be right but this is not how it behaves currently. I guess no one would mind if you rewrite it to behave like is_leap_version_at_least *but* revisit all locations where it's used currently